tests/testCoordinateTransformations.py

```diff
@@ -20,76 +17,73 @@ def controlEquationOfEquinoxes(mjd):
     D = JD - 2451545.0
     omegaDegrees = 125.04 - 0.052954*D
     Ldegrees = 280.47 + 0.98565*D
-    deltaPsiHours = -0.000319*numpy.sin(numpy.radians(omegaDegrees)) \
-                    - 0.000024 * numpy.sin(2.0*numpy.radians(Ldegrees))
+    deltaPsiHours = -0.000319*np.sin(np.radians(omegaDegrees)) \
+                    - 0.000024 * np.sin(2.0*np.radians(Ldegrees))
     epsilonDegrees = 23.4393 - 0.0000004*D
-    return (deltaPsiHours/24.0)*2.0*numpy.pi*numpy.cos(numpy.radians(epsilonDegrees))
+    return (deltaPsiHours/24.0)*2.0*np.pi*np.cos(np.radians(epsilonDegrees))
+
 
 def controlCalcGmstGast(mjd):
-    #From http://aa.usno.navy.mil/faq/docs/GAST.php Nov. 9 2013
+    # From http://aa.usno.navy.mil/faq/docs/GAST.php Nov. 9 2013
     mjdConv = 2400000.5
     jd2000 = 2451545.0
-    mjd_o = numpy.floor(mjd)
+    mjd_o = np.floor(mjd)
     jd = mjd + mjdConv
     jd_o = mjd_o + mjdConv
     h = 24.*(jd-jd_o)
     d = jd - jd2000
     d_o = jd_o - jd2000
     t = d/36525.
     gmst = 6.697374558 + 0.06570982441908*d_o + 1.00273790935*h + 0.000026*t**2
-    gast = gmst + 24.0*utils.equationOfEquinoxes(mjd)/(2.0*numpy.pi)
+    gast = gmst + 24.0*utils.equationOfEquinoxes(mjd)/(2.0*np.pi)
     gmst %= 24.
     gast %= 24.
     return gmst, gast
 
+
 class testCoordinateTransformations(unittest.TestCase):
 
     def setUp(self):
-        numpy.random.seed(32)
+        np.random.seed(32)
         ntests = 100
-        self.mjd = 57087.0 - 1000.0*(numpy.random.sample(ntests)-0.5)
+        self.mjd = 57087.0 - 1000.0*(np.random.sample(ntests)-0.5)
         self.tolerance = 1.0e-5
 
     def testExceptions(self):
         """
         Test to make sure that methods complain when incorrect data types are passed.
         """
         mjdFloat = 52000.0
-        mjd2 = numpy.array([52000.0, 53000.0])
-        mjd3 = numpy.array([53000.0, 53000.0, 54000.0])
```

Comment:
You are right.  It can take ints, now.  I added tests to make sure it runs when it is passed in an int and a float or two ints.